crates/orderbook/src/api/get_trades.rs

```diff
@@ -95,7 +95,7 @@ mod tests {
         assert_eq!(result.order_uid, None);
 
         let uid = OrderUid([1u8; 56]);
-        let order_uid_path = format!("/v1/trades?orderUid={:}", uid);
+        let order_uid_path = format!("/v1/trades?orderUid={uid:}");
```

Comment:
Does `{:}` even do anything special? Same for the change under this one.

Reply:
It doesn't and it already didn't before this PR. Will fix it in a follow up because I had auto merge on.